### DISCUSSION TITLE
Any possibility for a Flatpak?

### DISCUSSION THREAD

**Discussion by cgoetzke** [dundee/gdu#37] [Ideas]
Would you consider packaging gdu as a Flatpak?

**Comment by dundee** [comment]
I was considering Flatpak, but I don't see a big match here.

Flatpak is not very suitable for CLI tools, Snap is much better for this.

See https://www.reddit.com/r/linuxquestions/comments/6xtpjj/is_flatpak_suitable_for_cli_applications/

**Reply by cgoetzke** [reply]
Huh. I didn't know Flatpak doesn't work well with CLI tools. Thanks for educating me.

I don't like how Canonical keeps Snaps locked down only to their Snap Store. It feels too walled garden. But that's just me. It's too bad Flatpak doesn't even see this as an important issue. After reading through https://github.com/flatpak/flatpak/issues/1188, now I kind of don't like Snap or Flatpak.

Anyway, thanks for your time!
